.github/find-hotfix-tags.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+tag_regex='v[0-9]+\.[0-9]+\.[0-9]+-osghotfix-[A-Za-z]+'
+git tag -l | sort | egrep -x "$tag_regex" > git_tags
+
+token=$(curl -s \
+             -H "Content-Type: application/json" \
+             -X POST \
+             -d '{"username": "'${DOCKER_USERNAME}'", "password": "'${DOCKER_PASSWORD}'"}' \
```

Comment:
So i took another look at this and realized that the docker username/password shell variables are not actually quoted here.  Presumably that will be kind of ok for the username, but if the password has spaces the shell won't handle it right.

You can add quoting at least to handle spaces fairly easily:

```suggestion
             -d '{"username": "'"${DOCKER_USERNAME}"'", "password": "'"${DOCKER_PASSWORD}"'"}' \
```

Although if you want to be honest about it, it should not just be quoted for the shell, but potentially it needs to have proper json quoting (the most obvious example is if the password contains `\` or `"` characters).

So to be careful about it you could do something terrible like:

```suggestion
             -d "$(python <<-EOF
	import json
	import os
	print json.dumps({"username": os.getenv("DOCKER_USERNAME"),
	                  "password": os.getenv("DOCKER_PASSWORD")})
EOF
)" \
```

... Or shove that into a variable first (`json_auth=$(...)`) and then reference it in the curl command line (`-d "$json_auth").

Kinda makes me wish a tool like `jq` existed that let you turn command arguments into json output.

Kinda makes me want to write one...